src/BudgetManager/BudgetManager.SDK/DTOs/AddIncomeTransactionDTO.cs

```diff
@@ -8,7 +8,7 @@ public class AddIncomeTransactionDTO
 
         public Guid CategoryId { get; set; }
 
-        public DateTime DateOfTransaction { get; set; }
+        public DateTime DateOfTransaction { get; set; } = DateTime.UtcNow;
```

Comment:
I think user may add yesterdays or old transactions if has forgotten or didn't have time,

Reply:
it's just a default value. If id DTO we will not receive any information about DateOfTransaction - it would be DateTimeNow(moment when it's creating)